src/sql/platform/carbonEditLocalization/common/carbonEditStrings.ts

```diff
@@ -0,0 +1,74 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
We shouldn't be using the name "carbon" anywhere - that's a very old codename. 

I'd just put this under sql\base\common - no reason we need to make a separate folder for it.

Reply:
Will do,